home-data/src/main/java/com/softserveinc/ita/homeproject/model/Role.java

```diff
@@ -0,0 +1,43 @@
+package com.softserveinc.ita.homeproject.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
```

Comment:
Please don't use import via *. It is a "holivar" actually but please refer to https://stackoverflow.com/a/14897485